### PR TITLE
feat: dashboard showing data ingestion breakdown

### PR DESCRIPTION
**Pull request by sri-shetty**
# Summary

A concise description of the changes being introduced. If you are submitting a PR for a
new quickstart please review the pre merge checklist section to validate this PR is ready for review.

## Pre merge checklist

- [ ] Did you check you NRQL syntax? - Does it work?
- [ ] Did you check your dashboard image quality? -  Do they look good?
- [ ] Did you check that your alerts actually work?
- [ ] Did you include an InstallPlan and Documentation reference?
- [ ] Did you check your descriptive content for voice, tone, spelling and grammar errors?
- [ ] Did you attach images of your dashboards to the PR so we can see them working?

### Screenshots

Attach images of any visual changes, such as a dashboard here.

### CODE REVIEW

File: quickstarts/dataingestion/dashboards/data_ingestion_breakdown.json [hunk 1]
```diff
@@ -0,0 +1,986 @@
+{
+    "name": "Data Ingestion Breakdown",
+    "description": "",
+    "permissions": "PUBLIC_READ_WRITE",
```
Comment (Kav91) [inline quickstarts/dataingestion/dashboards/data_ingestion_breakdown.json:4]: Permissions attribute needs to be removed @sri-shetty, same issue that I experienced.
cc/ @rudouglas @polfliet @aswanson-nr

Comment (rudouglas) [reply quickstarts/dataingestion/dashboards/data_ingestion_breakdown.json:4]: yes this should pass after removing this line
```suggestion
```

Comment (sri-shetty) [reply quickstarts/dataingestion/dashboards/data_ingestion_breakdown.json]: Thank you @rudouglas and @Kav91 - Pushed PR again with the suggested changes.

File: quickstarts/account-data-ingestion/config.yml [hunk 2]
```diff
@@ -0,0 +1,37 @@
+# Name of the quickstart (required)
+name: acccount data ingestion
```
Comment (rudouglas) [inline quickstarts/account-data-ingestion/config.yml:2]: This should be:
```suggestion
name: account-data-ingestion
```

File: quickstarts/account-data-ingestion/config.yml [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+# Authors of the quickstart (required)
+authors:
+  - New Relic
+  - Sri Shetty
+
+# Keywords for filtering / searching criteria in the UI
+keywords:
+  - insights
+  - Data
+  - Ingest
```
Comment (rudouglas) [inline quickstarts/account-data-ingestion/config.yml:27]: I would remove insights here as it's a legacy product, there is a list of allowed keywords [here](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#keywords) but i think both `data` and `ingest` would be worth adding here
```suggestion
keywords:
  - data
  - ingest
```

File: quickstarts/account-data-ingestion/config.yml [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+# Reference to install plans located under /install directory
+# Allows us to construct reusable "install plans" and just use their ID in the quickstart config
+installPlans:
+  - teemetry-data-platform
```
Comment (rudouglas) [inline quickstarts/account-data-ingestion/config.yml:32]:
```suggestion
```
should be `telemetry-data-platform` i assume but...
I don't think this is one of our predefined [install plans](https://github.com/newrelic/newrelic-quickstarts/tree/main/install), is something required to be installed here to push data to the dashboard?

**Review by rudouglas — CHANGES_REQUESTED** [review]

File: quickstarts/account-data-ingestion/config.yml (hunk 2 above)
Comment (sri-shetty) [reply quickstarts/account-data-ingestion/config.yml]: Done

File: quickstarts/account-data-ingestion/config.yml (hunk 3 above)
Comment (sri-shetty) [reply quickstarts/account-data-ingestion/config.yml]: Done

File: quickstarts/account-data-ingestion/config.yml (hunk 4 above)
Comment (sri-shetty) [reply quickstarts/account-data-ingestion/config.yml]: Removed this section

File: quickstarts/account-data-ingestion/config.yml [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+
+# Authors of the quickstart (required)
+authors:
+  - New Relic
+  - Sri Shetty
+
+# Keywords for filtering / searching criteria in the UI
+keywords:
+  - Data
+  - Ingest
```
Comment (rudouglas) [inline quickstarts/account-data-ingestion/config.yml:26]:
```suggestion
  - data
  - ingest
```

**Review by rudouglas — APPROVED** [review]
Looks good to me

**Pull request merged**